Archives/VolFile.cpp

```diff
@@ -9,6 +9,14 @@
 
 namespace Archives
 {
+	// Volume section header tags
+	const std::array<char, 4> TagVOL_{ 'V', 'O', 'L', ' ' }; // Volume file tag
+	const std::array<char, 4> TagVOLH{ 'v', 'o', 'l', 'h' }; // Header tag
+	const std::array<char, 4> TagVOLS{ 'v', 'o', 'l', 's' }; // Filename table tag
+	const std::array<char, 4> TagVOLI{ 'v', 'o', 'l', 'i' }; // Index table tag
+	const std::array<char, 4> TagVBLK{ 'V', 'B', 'L', 'K' }; // Packed file tag
```

Comment:
I can live with this. The tags are being initialized by the actual string data. Uses are name checked so there are no typos.

I wonder if there might be a way to improve the intialization syntax using the StackOverflow answer from [How to initializez a `std::array<char, n>` with a string literal, omitting the trailing zero](https://stackoverflow.com/questions/33484233/how-to-initialize-a-stdarraychar-n-with-a-string-literal-omitting-the-trail). Admittedly I don't entirely understand the answer, though it looks short enough to study and figure out.

Reply:
I have split feelings about this. I wish there was a built in language solution to this problem. For these short arrays, it isn't a big deal though it would be nice to improve the syntax.

Reply:
Let's leave this for another day then. It may involve some learning and experimentation to come up with a satisfactory solution. I don't want to hold up the other changes on this.